src/FormModelInterface.php

```
@@ -76,62 +89,90 @@ public function getErrors(bool $first = false): array;
-     * @param bool $first Whether to return only the first error of each property.
+     * @param array $onlyFields Fields to include. Uses all fields when the list is empty.
+     * @param bool $first Whether to return only the first error of each field.
      *
      * @return array Flat list of error messages.
      *
-     * @phpstan-param list<string> $onlyProperties
+     * @phpstan-param list<string> $onlyFields
      * @phpstan-return array<int|string, string>
      */
-    public function getErrorSummary(array $onlyProperties = [], bool $first = false): array;
+    public function getErrorSummary(array $onlyFields = [], bool $first = false): array;
```

PHPMD (2.15.0)

[error] The method getErrorSummary has a boolean flag argument $first, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)